assets/src/edit-story/app/rightClickMenu/provider.js

```diff
@@ -363,6 +409,67 @@ function RightClickMenuProvider({ children }) {
     [handleCopyPage, menuItemProps, handleDeletePage, handlePastePage]
   );
 
+  const backgroundMediaItems = useMemo(
+    () => [
+      ...defaultItems,
+      {
+        label: RIGHT_CLICK_MENU_LABELS.DETACH_IMAGE_FROM_BACKGROUND,
+        separator: 'top',
+        onClick: handleRemoveMediaFromBackground,
+        ...menuItemProps,
+      },
+      {
+        label: RIGHT_CLICK_MENU_LABELS.REPLACE_BACKGROUND_IMAGE,
+        onClick: noop,
+        ...menuItemProps,
+      },
```

Comment:
Doing this one in a different pr. This is functionality that doesn't exist yet. It seems tricky so it'd be best to separate it out.